avm/res/container-instance/container-group/main.bicep

```diff
@@ -284,8 +284,8 @@ type containerType = {
         @description('Optional. The GPU request of this container instance.')
         gpu: containerGpuType?
 
-        @description('Optional. The memory request in GB of this container instance. To specify a decimal value, use the json() function.')
-        memoryInGB: int?
+        @description('Optional. The memory request in GB of this container instance.')
+        memoryInGB: string?
```

Comment:
The resource documentation says

> memoryInGB 
>
> The memory limit in GB of this container instance. To specify a decimal value, use the json() function. 
>
> int or json decimal

https://learn.microsoft.com/en-us/azure/templates/microsoft.containerinstance/containergroups?pivots=deployment-language-bicep#resourcelimits

I'm not sure whether this might lead to misunderstandings?

might be good to use type `any` and add the description again?

@eriqua, @AlexanderSehr what do you think?

Reply:
> The resource documentation says
> 
> > memoryInGB
> > The memory limit in GB of this container instance. To specify a decimal value, use the json() function.
> > int or json decimal
> 
> https://learn.microsoft.com/en-us/azure/templates/microsoft.containerinstance/containergroups?pivots=deployment-language-bicep#resourcelimits
> 
> I'm not sure whether this might lead to misunderstandings?
> 
> might be good to use type `any` and add the description again?
> 
> @eriqua, @AlexanderSehr what do you think?

@Azure/avm-core-team I need a second pair of eyes.